specification/dataprotection/resource-manager/Microsoft.DataProtection/preview/2021-06-01-preview/examples/VaultCRUD/CheckBackupVaultsNameAvailability.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
updated all instances of version 2021-02-01-preview to 2021-06-01-preview in file.